lib/connection/statement.js

```diff
@@ -359,6 +359,7 @@ function createContextPreExec(
   statementContext.multiCurId = statementOptions.multiCurId;
   statementContext.rowMode = statementOptions.rowMode;
   statementContext.asyncExec = statementOptions.asyncExec;
+  statementContext.cwd = statementOptions.cwd;
```

Comment:
Can we validate cwd argument before adding to statementContext?

Reply:
Okay!